--- nagios-plugin-check_snmp_load-1.2/check_snmp_load.pl~	2006-02-14 14:53:36.000000000 +0200
+++ nagios-plugin-check_snmp_load-1.2/check_snmp_load.pl	2006-02-14 14:54:42.000000000 +0200
@@ -18,7 +18,7 @@
 
 # Nagios specific
 
-use lib "/usr/local/nagios/libexec";
+use lib "@plugindir@";
 use utils qw(%ERRORS $TIMEOUT);
 #my $TIMEOUT = 15;
 #my %ERRORS=('OK'=>0,'WARNING'=>1,'CRITICAL'=>2,'UNKNOWN'=>3,'DEPENDENT'=>4);
